django/template/base.py

```diff
@@ -988,8 +988,7 @@ def render(self, context, stream=False):
     def _stream(self, context, stream=True):
         for node in self:
             if isinstance(node, Node):
-                for bit in node.stream_annotated(context):
-                    yield mark_safe(str(bit)) if stream else str(bit)
+                yield from map(lambda bit: mark_safe(str(bit)) if stream else str(bit), node.stream_annotated(context))
```

Comment:
Wouldn't `(mark_safe(str(bit)) if stream else str(bit) for bit in node.stream_annotated(context))` be better here? To avoid the map/lambda combo.

Reply:
made changes